### PR TITLE
Added Lucas to 'about us'

### PR DESCRIPTION
**Pull request by LuSilvaDeveloper**
|                                    Web Dev Path                                    |
| :--------------------------------------------------------------------------------: |

#### Have you updated the CHANGELOG.md file? If not, please do it.

Yes.

#### What is this change?

Added Lucas to the About page.

#### Were there any complications while making this change?

Yes. I was facing some problems with the picture. I feel it is a little stretched, but I'm not 100% sure.

#### How did you verify this change?

Locally With different browsers.

#### When should this be merged?

After review.

### CODE REVIEW

**Review by cherylli — APPROVED** [review]
Looks good

**Review by Satoshi-Sh — APPROVED** [review]
Looks good to me. I think your image is displayed fine.

**Review by mariana-caldas — APPROVED** [review]
Other than the content corrections, it looks good to me. Thanks, @LuSilvaDeveloper!

File: utils/about.js [hunk 1]
```diff
@@ -69,7 +69,16 @@ export const whoWeAre = [
...
   },
-
+  {
+    image: '/images/lucas.webp',
+    name: 'Lucas Silva',
+    title: '',
+    position: 'Full-Stack Developer',
+    linkedIn: 'lucas-assuncao-silva',
+    portfolio: '',
+    about:
```
[the remark this answers is not in the document]
Comment (LuSilvaDeveloper) [reply utils/about.js:79]: Nice! Fixed. Thank you so much @mariana-caldas

File: components/containers/Member/styles.js [hunk 2]
```diff
@@ -93,7 +93,7 @@ const ImageWrapper = styled.div`
 
   img {
     border-radius: 50%;
-    objectfit: 'cover';
+    object-fit: cover;
   }
```
[the remark this answers is not in the document]
Comment (LuSilvaDeveloper) [reply components/containers/Member/styles.js:97]: Thanks for @cherylli ! She noticed that!

**Pull request merged**